kitchen.dokken.yml

```diff
@@ -2,7 +2,7 @@
 driver:
   name: dokken
   privileged: true
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  chef_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
```

Comment:
I believe we've been using `current in most other cookbooks, is there a reason to change this here?

Reply:
I wouldn't think so it's probably a copy/paste-o